content/uk/docs/reference/glossary/index.md

```diff
@@ -0,0 +1,17 @@
+---
+approvers:
+- chenopis
+- abiogenesis-now
```

Comment:
nit: remove these approvers unless you're confident that they speak Ukrainian

Reply:
Do we need replace them by us, or just remove?